Create ui-c8y-1023-97-0-re-enable-duplicating-global-smart-rules-in-local-contexts-12280-graft.md (#4858)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-97-0-re-enable-duplicating-global-smart-rules-in-local-contexts-12280-graft.md

---
date: ""

> title: Re-enabled duplication of global smart rules in local contexts

Reviewer: ```suggestion
title: Restored option to duplicate global smart rules in local contexts
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-67052
version: 1023.97.0
---

> An earlier update to smart rule management options accidentally disabled duplication of global smart rules into local contexts. This fix restores that capability, so global smart rules can again be duplicated and applied in the context of specific groups and devices.

Reviewer: ```suggestion
An earlier update to smart rule management options removed the option to duplicate global smart rules in local contexts. This fix restores that capability as a "Duplicate as local" option in the smart rule's drop down menu, so global smart rules can again be duplicated and applied in the context of specific group or device.
```

> An earlier update to smart rule management options removed the option to duplicate global smart rules in local contexts. This fix restores that capability as a "Duplicate as local" option in the smart rule's drop down menu, so global smart rules can again be duplicated and applied in the context of specific group or device.

Reviewer 2: ```suggestion
An earlier update to the smart rule management options removed the option to duplicate global smart rules in local contexts. This fix restores that capability as a **Duplicate as local** option in the smart rule's dropdown menu, so global smart rules can again be duplicated and applied in the context of a specific group or device.
```

---
date: ""

> title: Re-enable duplicating global smart rules in local contexts (#12280) [GRAFT][release/cd] (#12352)

Reviewer: ```suggestion
title: Re-enabled duplication of global smart rules in local contexts
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP

>     label: Feature

Reviewer: ```suggestion
  - value: change-VSkj2iV9m
    label: Fix
```

component:
  - value: component-YbYJ3gLU_

>     label: Web SDK

Reviewer: ```suggestion
  - value: component-YdSEScrEC
    label: Cockpit
```

build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-67052
version: 1023.97.0
---
# Backport

This will backport the following commits from `develop` to `release/cd`:
- [feat(Web SDK): [MTM-67052] Re-enable duplicating global smart rules
in local contexts
(#12280)](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/12280)

<!--- Backport version: 12.0.4 -->

### Questions ?
Please refer to the [Backport tool
documentation](https://github.com/sorenlouv/backport)

[MTM-67052]:

> https://cumulocity.atlassian.net/browse/MTM-67052?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Reviewer: ```suggestion
An earlier update to smart rule management options accidentally disabled duplication of global smart rules into local contexts. This fix restores that capability, so global smart rules can again be duplicated and applied in the context of specific groups and devices.
```
